Fitness/Views/ProBioPageViewController.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  ProBioPageViewController.swift
+//  
+//
+//  Created by Cameron Hamidi on 2/28/19.
+//
+
+import UIKit
+
+class ProBioPageViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    var backButton: UIButton!
```

Comment:
alphabetizing these variables would be nice.

Reply:
Will do

Reply:
also make them `private`